docs/guide/pricing-guide.md

```diff
@@ -0,0 +1,61 @@
+title: Pricing Guide
+description: This guide explains how Sematext Cloud's flexible and transparent pricing works, helping you manage costs effectively based on your needs.
+
+We offer flexible and transparent pricing for all of our four solutions: [Monitoring](https://sematext.com/docs/monitoring/), [Logs](https://sematext.com/docs/logs/), [Experience](https://sematext.com/docs/experience/), and [Synthetics](https://sematext.com/docs/synthetics/). This guide explains how our pricing works, helping you manage costs effectively based on your needs.
+
+## Apps and Plans
+For each of the four solutions, you can create multiple Apps based on your needs. Each App can be tailored to your specific requirements with different plans. This allows you to customize and control costs for each App individually.
+
+- **Logs Apps**: Offer three different plan options with different retention periods and daily log volumes.
```

Comment:
I'm looking at the raw content, not rendered, but I have a feeling that a bulleted list for each solution would be more readable, with each bullet listing one factor that affects the cost for that solution.